cpp/downgrades/a8c2176e9a5cf9be8d17053a4c8e7e56b5aced6d/upgrade.properties

```
@@ -0,0 +1,3 @@
+description: Support `__leave` statement
+compatibility: backwards
```

Since we get the exact database we would get if we would have run the only version of the extractor.
```suggestion
compatibility: full
```